### PR TITLE
Remove Build and CI

### LINKED ISSUE #119: Remove CI and build support
Build and CI support complicate zazu and have good reasons to live in a spearate tool.

### PR DESCRIPTION
**Pull request by stopthatcow**
This may be controvertial but I think removing CI and build support will streamline zazu by drastically simplifying the config file. There were scant support for much besides cmake and build scripts. Without build support CI support doesn't make much sense.

Fixes #119

### CODE REVIEW

**Comment by stopthatcow** [comment]
@moorepatrick @aramnhammer I'd like your feedback on this. It removes functionality. I am not sure anyone is actually using this though.

File: zazu.yaml [hunk 1]
```diff
@@ -1,17 +1,4 @@
...
-          - arch: python
-            script:
-              - rm -rf dist &&
-                python setup.py build_sphinx bdist_wheel
-      - name: coverage
-        builds:
-          - arch: python
-            script:
-              - tox
+# Zozu config file for zazu itself.
```
Comment (moorepatrick) [inline zazu.yaml]: ⚡️  **_Spellcheck_** ⚡️ 

`Zazu`?

**Review by moorepatrick — COMMENTED** [review]
One spelling mistake, otherwise no comments.

File: zazu.yaml (hunk 1 above)
Comment (stopthatcow) [reply zazu.yaml]: Good eye!

**Pull request merged**